astroquery/esa/euclid/tests/test_euclidtap.py

```diff
@@ -1084,6 +1084,13 @@ def test_get_spectrum_exceptions():
 
     assert str(exc_info.value).startswith('Missing required argument')
 
+    with pytest.raises(Exception) as exc_info:
```

Comment:
Always be as specific as possible

```suggestion
    with pytest.raises(ValueError) as exc_info:
```